backend/src/apiserver/main.go

```diff
@@ -61,7 +63,14 @@ func main() {
 
 	initConfig()
 	clientManager := newClientManager()
-	resourceManager := resource.NewResourceManager(&clientManager)
+	resourceManager := resource.NewResourceManager(
+		&clientManager,
+		map[string]interface{}{
+			"CollectMetrics":   *collectMetricsFlag,
+			"ApiVersion":       *apiVersion,
+			"DefaultNamespace": *defaultNamespace,
+		},
```

Comment:
Good catch. I forgot to remove these. The only one required is the default namespace. We can remove the generic `map[string]interface{}` too. 

The reason for the `defaultNamespace` is because resources are namespaced and implicit use of an empty string may become a problem as this is the default value when we convert to `model` types. Primarily, we use it in single-user mode, but this can be extended to multi-user mode as a "common" namespace for all users (this not a part of this PR and does not block v1beta1 GA).